src/main/groovy/wooga/gradle/paket/unity/PaketUpmPackageSpec.groovy

```diff
@@ -0,0 +1,68 @@
+package wooga.gradle.paket.unity
+
+import com.wooga.gradle.BaseSpec
+import org.gradle.api.provider.MapProperty
+import org.gradle.api.provider.Property
+import org.gradle.api.provider.Provider
+import org.gradle.api.tasks.Input
+import org.gradle.api.tasks.Optional
+
+trait PaketUpmPackageSpec implements BaseSpec {
+
+    /**
+     * Map [<paket_package_name>: <upm_manifest_contents>] that overrides the contents of generated package.json files in UPM package mode.
+     */
```

Comment:
The comment is on the private backing field. It should be on the public property, no?